### ISSUE TITLE
Should this repo be private?

### ISSUE THREAD

**Issue by sirosen** [globusonline/docs.globus.org#223]
Not that we should _hide_ it, but it's not a project on which we're soliciting contributions or anything.
If we're wiring up Jenkins, I start to prefer it being private. I don't love having an open project with links to Jenkins because we're not about to open up any kind of public access to view Jenkins build results (it's just a lot easier to have Jenkins locked down).
Part of the worry here is just having to deal with potential confusion and requests about this.

@teresas, I don't think any part of our process for this repo relies on it being public, right?
I assume it was public because we used to use GitHub Pages.

cc @stumartin, there may be a policy question here too.

**Comment by teresas** [comment]
I agree we can and maybe should change it to private. I believe you're right about it being public because we were using GitHub Pages, but since we don't anymore it should be ok to switch.
